javascript-arrays/main.js

```diff
@@ -0,0 +1,21 @@
+// A new array literal is being assigned to 'colors' variable
+const colors = ['red', 'white', 'blue'];
+
+console.log('value of colors[0]:', colors[0]); // the log method of the console object is being called with two arguments a string and 'colors' element at 0
+console.log('value of colors[1]:', colors[1]); // the log method of the console object is being called with two arguments a string and 'colors' element at 1
+console.log('value of colors[2]:', colors[2]); // the log method of the console object is being called with two arguments a string and 'colors' array element at 2
+console.log('America is', colors[0] + ', ' + colors[1] + ', ' + colors[2]); // the log method of the console object is being called with two arguments a sting and a concatnated string
```

Comment:
Missing `'and'` & a `'.'`. The string should be "America is red, white, and blue."